metadata/src/test/java/org/apache/kafka/metalog/LocalLogManager.java

```diff
@@ -198,7 +207,12 @@ synchronized void unregisterLogManager(LocalLogManager logManager) {
             }
         }
 
-        synchronized long tryAppend(int nodeId, long epoch, LocalBatch batch) {
+        synchronized long tryAppend(int nodeId, int epoch, List<ApiMessageAndVersion> batch) {
+            // No easy access to the concept of time. Use the base offset as the append timestamp
```

Comment:
Can you multiply the offset by 10 or something so that the numbers for time are slightly different than those for offset? That would help ensure we're not getting them mixed up elsewhere in the code.

Reply:
Good idea. Done.